dasharo-compatibility/logo-customization-functionality.robot

```diff
@@ -21,6 +21,8 @@ Resource            ../keys.robot
 # exactly the case right now)
 Suite Setup         Run Keywords
 ...                     Prepare Test Suite
+...                     AND
+...                     Make Sure That Lock The BIOS Boot Medium Is Disabled
```

Comment:
There is one more lock that prevents the platform to be flashed internally: dasharo-security/smm-bios-write-protection.robot